near-sdk/src/json_types/vector.rs

```diff
@@ -0,0 +1,83 @@
+/// Helper class to serialize/deserialize `Vec<u8>` to base64 string.
+use serde::de::Error;
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct Base64VecU8(pub Vec<u8>);
```

Comment:
Please add docstring here. If there is a repetition between the module-level docstring and the struct-level docstring, use the latter one.